fedimint-rocksdb/src/lib.rs

```diff
@@ -24,6 +24,21 @@ use tracing::debug;
 
 use crate::envs::FM_ROCKSDB_WRITE_BUFFER_SIZE_ENV;
 
+// turn an `iter` into a `Stream` where every `next` is ran inside
+// `block_in_place` to offload the blocking calls
```

Comment:
block_in_place doesn't offload anything

Reply:
Right. But it offloads other async code to the other executor threads. :D